src/index.html

```diff
@@ -8,11 +8,49 @@
     />
     <title>Calendar</title>
     <link
-      rel="stylesheet"
-      href="styles/index.scss"
+      rel="styles"
```

Comment:
The `rel` attribute should be `stylesheet` not `styles` to properly link the CSS file.